### PR TITLE
Fix issue#8440

### LINKED ISSUE #8440: Add test case for UpdateResponseHeader

### PR DESCRIPTION
**Pull request by huanghao495430759**
Fixes #8440 .

Add test case for UpdateResponseHeader #8440

### CODE REVIEW

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/header/update/UpdateResponseHeaderTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+public final class UpdateResponseHeaderTest {
+
+    @Test
+    public void assertPropertiesWhenExecuteResultOfEmptyList() {
+        UpdateResponseHeader updateResponseHeader = new UpdateResponseHeader(mock(SQLStatement.class));
+        assertThat(updateResponseHeader.getLastInsertId(), is(0L));
+        //before mergeUpdateCount
```
Comment (terrymanu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/header/update/UpdateResponseHeaderTest.java:38]: Please remove unnecessary comment

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/header/update/UpdateResponseHeaderTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        assertThat(updateResponseHeader.getUpdateCount(), is(1L));
+        updateResponseHeader.mergeUpdateCount();
+        //after mergeUpdateCount
+        assertThat(updateResponseHeader.getUpdateCount(), is(4L));
+    }
+
+    private Collection<ExecuteResult> getExecuteUpdateResults() {
+        UpdateResult updateResult1 = new UpdateResult(1, 2L);
+        UpdateResult updateResult2 = new UpdateResult(3, 4L);
+
```
Comment (terrymanu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/header/update/UpdateResponseHeaderTest.java:59]: Please remove unnecessary blank lines

**Review by terrymanu — CHANGES_REQUESTED** [review]

**Review by huanghao495430759 — COMMENTED** [review]
remove unnecessary comment and blank line.

**Review by terrymanu — CHANGES_REQUESTED** [review]
Please fix the CI error

**Review by tristaZero — APPROVED** [review]
Hi @huanghao495430759 
Thanks for your job. :)

**Review by terrymanu — APPROVED** [review]

**Pull request merged**